docs_dev/releasing.md

```diff
@@ -342,6 +347,10 @@ You can create a release branch via the GitHub UI.
 
 A release candidate is a tag of the form `v${MAJOR}.${MINOR}.${PATCHLEVEL}-rc.${N}`, where `N` is a small integer, and a release candidate tag always points to a commit on the corresponding minor release branch.  Push this tag to GitHub and announce a release vote on kubeflow-discuss.
 
+### Update Version Shown on Central Dashboard
+
+Update release tag to central dashboard: [link](https://github.com/kubeflow/manifests/blob/eab125dd42c08b0e2495b7b2d7a7010ac64d3774/application/application/overlays/application/application.yaml#L16)
```

Comment:
Can you clarify this section; we aren't updating the central dashboard.
We are updating the version specified in the Kubeflow application resource 
(the fact that central dashboard is displaying that is a secondary effect).